src/rprblender/nodes/blender_nodes.py

```diff
@@ -1254,6 +1254,15 @@ def export(self):
                 res = (in1 / in2).floor() * in2
             elif op == 'ABSOLUTE':
                 res = abs(in1)
+            elif op == 'SCALE':
+                # input 2 here is a scalar
+                res = in1 * in2
+            elif op == 'SINE':
+                res = in1.sin()
+            elif op == 'COSINE':
+                res = in1.cos()
+            elif op == 'TANGENT':
+                res = in1.tan()
```

Comment:
One value operations like sin. cos, tan better to move to 1214 line (before else section). Current section relates to 2-values operations.
Also would be good to move there other one-value operations like: FLOOR, CEIL, LENGTH, ABSOLUTE